addons/repair/views/repair_views.xml

```diff
@@ -56,23 +58,21 @@
                             </div>
                         </button>
                     </div>
-                    <label for="name"/>
-                    <h1>
-                        <field name="name"/>
-                    </h1>
                     <group>
                         <group>
+                            <field name="description"/>
                             <field name="invoice_state" invisible="1"/>
                             <field name="tracking" invisible="1" attrs="{'readonly': 1}"/>
                             <field name="product_id" />
+                            <field name="lot_id" context="{'default_product_id': product_id, 'default_company_id': company_id}" groups="stock.group_production_lot" attrs="{'required':[('tracking', 'in', ['serial', 'lot'])], 'readonly': ['|', ('state', '=', 'done'), ('tracking', 'not in', ['serial', 'lot'])]}"/>
                             <field name="product_uom_category_id" invisible="1"/>
                             <label for="product_qty"/>
                             <div class="o_row">
                                 <field name="product_qty" attrs="{'readonly':[('tracking', 'in', ['serial'])]}"/>
                                 <field name="product_uom" groups="uom.group_uom"/>
                             </div>
-                            <field name="lot_id" context="{'default_product_id': product_id, 'default_company_id': company_id}" groups="stock.group_production_lot" attrs="{'required':[('tracking', 'in', ['serial', 'lot'])], 'invisible': [('tracking', 'not in', ['serial', 'lot'])], 'readonly': [('state', '=', 'done')]}"/>
                             <field name="partner_id" widget="res_partner_many2one" attrs="{'required':[('invoice_method','!=','none')]}" context="{'res_partner_search_mode': 'customer', 'show_vat': True}"/>
+                            <field name="sale_order_id"/>
```

Comment:
in the current form, if you Modify the `partner_id`, it modifies  `address_id` (onchange). It is weird to have a sale order fields between these 2 fields. IMO, it is more convient to put this fields after `address_id` of before `partner_id` but not between.